### PR TITLE
Formatting tools and reformat code repo

### PR DESCRIPTION
**Pull request by keptsecret**
On further inspection, saw that none of the files actually used MSVC format anyways and were all different, so used slightly modified Google format

### CODE REVIEW

**Review by elsien-dev — APPROVED** [review]

File: .clang-format [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+BreakBeforeBinaryOperators: NonAssignment
+BreakBeforeBraces: Custom
+BreakInheritanceList: AfterColon
+BreakBeforeTernaryOperators: true
+BreakConstructorInitializers: BeforeComma
+BreakStringLiterals: false
+ColumnLimit: 0
+ConstructorInitializerAllOnOneLineOrOnePerLine: true
+Cpp11BracedListStyle: false
+EmptyLineBeforeAccessModifier: LogicalBlock
```
Comment (kevinsadi) [inline .clang-format]: remove this line 

-EmptyLineBeforeAccessModifier: LogicalBlock

File: .clang-format [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+IndentCaseBlocks: false
+IndentCaseLabels: true
+IndentPPDirectives: BeforeHash
+IndentWidth: 4
+NamespaceIndentation: All
+PointerAlignment: Left
+SortIncludes: false
+SpaceBeforeCaseColon: false
```
Comment (kevinsadi) [inline .clang-format]: remove this line 

-SpaceBeforeCaseColon: false

**Review by kevinsadi — CHANGES_REQUESTED** [review]
Video to set up clang-format on Visual Studio Code -> https://www.youtube.com/watch?v=rd8-aVc3Krg

There are two lines that seemingly do not work on my version of clang (which I thought was the newest) on Visual Studio Code. I figure it is simplest to just remove these two rules that do not work.

**Review by kevinsadi — APPROVED** [review]
LGTM

**Pull request merged**